src/debug.c

```diff
@@ -1968,13 +1987,31 @@ void sigsegvHandler(int sig, siginfo_t *info, void *secret) {
 
 #ifdef HAVE_BACKTRACE
     ucontext_t *uc = (ucontext_t*) secret;
-    void *eip = getMcontextEip(uc);
+    void *eip = getAndSetMcontextEip(uc, NULL);
     if (eip != NULL) {
         serverLog(LL_WARNING,
         "Crashed running the instruction at: %p", eip);
     }
 
-    logStackTrace(getMcontextEip(uc), 1);
+    if (eip == info->si_addr) {
+        /* We crashed when calling a none mapped pointer.
+         * Set pointer to some valid address otherwise logStackTrace will crash
+         * when tryint to access this pointer. */
+        serverLog(LL_WARNING, "Crashed caused by calling to invalid function pointer %p.", eip);
```

Comment:
```suggestion
        serverLog(LL_WARNING, "Crash probably due to an invalid function pointer %p.", eip);
```
do we really need this print? the `invalidFunctionWasCalled` be clearly visible in the stack trace, and the real EIP was already printed too.

Reply:
Agree, will remove it.